piker/ui/_exec.py

```diff
@@ -124,6 +149,40 @@ def closeEvent(
         # raising KBI seems to get intercepted by by Qt so just use the system.
         os.kill(os.getpid(), signal.SIGINT)
 
+    @property
+    def status_bar(self) -> QStatusBar:
+
+        # style and cached the status bar on first access
+        if not self._status_bar:
+            # TODO: i guess refactor stuff to avoid having to import here?
+            from ._style import _font, hcolor
+            sb = self.statusBar()
+            sb.setStyleSheet((
+                f"color : {hcolor('gunmetal')};"
+                f"background : {hcolor('default_dark')};"
+                # "min-height : 19px;"
+                f"font-size : {_font.px_size - 4}px;"
```

Comment:
Bar itself looks great, on low dpi monitors tho we have a font size issue, on my PC the correct font size setting would be: `font-size : {_font.px_size + 1}px;`

Reply:
kk, might need a small adhoc condition then to adjust when dpi is detected as 96ish.